frontend/assets/icons/viewer/align.svg.tsx

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
Clip icon should update to clip box if i selected a viewpoint with a clip box in view:
![24-01-04 02-32-09](https://github.com/3drepo/3drepo.io/assets/11945337/396971cb-edeb-4d28-b917-85d3622ae7d9)